tools/coreml/test/test_mxnet_converter.py

```diff
@@ -18,18 +18,16 @@
 import unittest
 import mxnet as mx
 import numpy as np
-import sys
-import os
-current_working_directory = os.getcwd()
-sys.path.append(current_working_directory + "/..")
-sys.path.append(current_working_directory + "/../converter/")
-import _mxnet_converter as mxnet_converter
+
+from converter._mxnet_converter import convert
```

Comment:
Does this use the pip installed version of the converter?  Would it make more sense to use the version that's in the codebase so that we can catch potential failures in that codebase when a new PR is opened?  LMK if I'm misunderstanding this.

Reply:
The mxnet-to-coreml converter is currently a standalone package on PyPI. It does not build together with MXNet.

Reply:
Oups, closed by accident

Reply:
+1, I don't think this is the right way to test. We should catch problems before a package is released